### ISSUE TITLE
Static server not working

### ISSUE THREAD

**Issue by huseeiin** [bunsvr/utils#1]
```js
import { Router } from "@stricjs/router";
import { dir } from "@stricjs/utils";

export default new Router().all("/public/*", dir("./public"));
```

Display's Bun's standard "Welcome to Bun!" page. No error in console either.

**Comment by aquapi** [comment]
Set `parsePath` to `true` solves the problem

I forgot to change `r.path` to `r.url` with wildcards when `parsePath` is not set in 4.1.0

Will be fixed in 4.1.1

**Comment by aquapi** [comment]
@xxxhussein I will release 4.1.1 and close the issue
